cmd/server/assets/login/register-phone.html

```diff
@@ -145,7 +216,9 @@
         user.multiFactor.enroll(multiFactorAssertion, $displayName.val()).then(function() {
           flash.clear();
           flash.alert('SMS authentication enrolled successfully.');
-          $skip.text("Continue")
+          $skip.text('Continue');
+          $registerDiv.show();
```

Comment:
I don't think this will work with d-none, but maybe jquery is smarter than me. I think you need to call `.removeClass`